### PR TITLE
Get the i18n build process up to date with develop

### PR DESCRIPTION
**Pull request by cea2aj**
Get the i18n build process up to date with develop

Update the translator to correspond with the code in jambo. Update the preprocessors to correspond to the latest translator changes. Include changes from this [PR](https://github.com/yext/answers/pull/1017) which didn't get included in the automatic merge.

J=SLAP-669
TEST=manual

Run a build and observes the properly formatted calls to ANSWERS.processTranslation() and the HBS processTranslation helper. Run an internationalized jambo build and test the internationalized builds. Observe the translations on the page.

### CODE REVIEW

File: conf/gulp-tasks/template/bundletemplatestaskfactory.js [hunk 1]
```diff
@@ -99,17 +97,26 @@ class BundleTemplatesTaskFactory {
...
-          {
-            include: `./dist/${precompiledFile}`
-          }),
         builtins(),
         commonjs({
           include: './node_modules/**'
         }),
         babel({
-          presets: ['@babel/env']
+          runtimeHelpers: true,
```
Comment (oshi97) [inline conf/gulp-tasks/template/bundletemplatestaskfactory.js:105]: I don't think we should be adding the polyfills back in

**Review by oshi97 — COMMENTED** [review]
where there any merge conflicts besides with the remove global polyfills pr?

**Comment by cea2aj** [comment]
> where there any merge conflicts besides with the remove global polyfills pr?

I believe there were just a couple in the package-lock.json

File: conf/gulp-tasks/template/bundletemplatestaskfactory.js (hunk 1 above)
Comment (cea2aj) [reply conf/gulp-tasks/template/bundletemplatestaskfactory.js:105]: I was just making sure that the changes from this commit got in here: https://github.com/yext/answers/pull/1017. Is that not what we want?

Comment (oshi97) [reply conf/gulp-tasks/template/bundletemplatestaskfactory.js:105]: sorry you're right, totally misread

**Pull request closed without merging**